### PR TITLE
How to become a maintainer : Add a link for creating the issue

### PR DESCRIPTION
**Pull request by ttoine**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | to improve the "how to become a maintainer" page?
| Fixed ticket? | no

### CODE REVIEW

File: src/content/1.7/project/maintainers-guide/how-to-become-a-maintainer.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ All the above, plus:
 
 ## How to apply to be a maintainer
 
-1. Open an issue on the open source repository explaining why you apply to become a maintainer, and on what part of the project (Core or module/subproject).
+1. [Open an issue on the open source repository](https://github.com/PrestaShop/open-source/issues) explaining why you apply to become a maintainer, and on what part of the project (Core or module/subproject).
```
Comment (Progi1984) [inline src/content/1.7/project/maintainers-guide/how-to-become-a-maintainer.md:45]:
```suggestion
1. [Open an issue on the open source repository](https://github.com/PrestaShop/open-source/issues/new) explaining why you apply to become a maintainer, and on what part of the project (Core or module/subproject).
```

**Review by Progi1984 — COMMENTED** [review]
A small improvement

**Comment by ttoine** [comment]
you think directly on the creation form is better?

**Comment by Progi1984** [comment]
@ttoine We ask to open an issue so the link comes to the form directly. Full logical.

**Comment by ttoine** [comment]
ok, let's do that.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**